spec/spec.md

```diff
@@ -2693,7 +2697,79 @@ This IPEX [[ref: IPEX]] protocol leverages important features of ACDCs and ancil
 |`grant`|  | Y | Full or Selective Disclosure ACDC, signature on `grant` or its SAID  | includes Attribute values, CESR-Proof signature |
 | | `admit` | N | signature on `grant` or its SAID  | CESR-Proof signature |
 
-#### Commitments via SAID
+When messages of these types are validly exchanged depends on messages previously exchanged and the resulting states of the disclosee and discloser, as described below.
+
+#### Discloser States and Transitions
+The following states and transitions are defined for the Discloser. These state names are non-normative; however, they are helpful for understanding the protocol.
+
+**States**
+
+| State | On Entry, Send to Disclosee | Description |
+|:--|:--|:--|
+| `Applied` |
+| `SpurningApply` | spurn |
+| `Offering` | offer | Prepare and send either a requested offer (referencing disclosee's apply) or unsolicited offer |
+| `Offered` |
+| `Agreed` |
+| `SpruningAgreed` | spurn |
```

Comment:
```suggestion
| `SpurningAgreed` | spurn |
```

Reply:
@lenkan Thanks for the spelling fix.  I'll make this in my local repo, which might be used for documenting keripy and vLEI ecosystem.